src/Tokenizers/PHP.php

```diff
@@ -1812,7 +1812,7 @@ protected function processAdditional()
                     }
                 }
 
-                if ($this->tokens[$x]['code'] === T_OPEN_PARENTHESIS) {
+                if (isset($this->tokens[$x]) === true && $this->tokens[$x]['code'] === T_OPEN_PARENTHESIS) {
```

Comment:
[isset](https://www.php.net/manual/en/function.isset.php) Always returns `bool`. `=== true` is redundant and should be removed.

Reply:
@tvanrielwendrich While that is true, the PHPCS native code style rules **demand** the `=== true`, so no, this cannot be removed.